### DISCUSSION TITLE
Name squatting for 9 years

### DISCUSSION THREAD

**Discussion by noahfraiture** [community/community#127877] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,

I'm an entrepreneur and use "noahcode" as my company name, I own the domain name noahcode.dev. However this isn't a trademark, just a name. 
I would like the github username noahcode but there's already a profile with this username which have no public activity for 9 years, there's only a hello world repo on this profile. I don't think there's even private activity. I made a github issue on his "hello world" public repo to contact him but I don't have any answer.

I saw that github contact doesn't take request to revoke username anymore. Is there something I can do ? This clearly seems to be a violation of the github policy of no username squatting

**Comment by ebndev** [comment]
Hi @noahfraiture,

As you mentioned, we are no longer accepting requests to release dormant usernames. I will be locking this discussion as we are not able to accommodate this request.

Thanks for understanding!
